src/taxes/components/TaxInput/styles.ts

```diff
@@ -0,0 +1,21 @@
+import { makeStyles } from "@saleor/macaw-ui";
+
+export const useStyles = makeStyles(
+  () => ({
+    hideSpinboxes: {
```

Comment:
OK, this is awesome, and I had no idea that I can do this 👏🏻 

Can we name it a little different, though? I had no idea what `hideSpinboxes` and those weird browser prefixes did - maybe `hideInputArrows`?

```suggestion
    hideInputArrows: {
```

Reply:
I can add a comment here, but I would keep the name because these input arrows are widely known as spin boxes or spin buttons across different platforms.